tests/Mocks/PlentyRequestMock.php

```diff
@@ -198,4 +198,11 @@ public function getQueryString(): string
     public function getMethod(): string
     {
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function getLocale(): string
+    {
+    }
```

Comment:
Is this related to the price filter? Or a change which sneaked in?

Reply:
I have updated the dependency `plentymarkets/plugin-interface`. Since the abstract class (which this one extends from) added this method additionally as `abstract` I also needed to add it here.